dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/TaskInstanceServiceImpl.java

```diff
@@ -171,6 +176,12 @@ public Result queryTaskListPaging(User loginUser,
             return result;
         }
 
+        TaskDefinition taskDefinition = taskDefinitionMapper.queryByCode(task.getTaskCode());
+        if (taskDefinition != null && projectCode != taskDefinition.getProjectCode()) {
+            putMsg(result, Status.TASK_INSTANCE_NOT_FOUND, taskInstanceId);
```

Comment:
is the message prompt ` Status.TASK_INSTANCE_NOT_FOUND` right?

Reply:
The purpose is to determine whether TaskInstance belongs to project but  project_code is not in t_ds_task_instance, now we have to query t_ds_task_definition to judge  whether TaskInstance belongs to project , if (taskDefinition != null && projectCode != taskDefinition.getProjectCode()) means taskInstanceId  does not belong to the project.